webapp/components/create_comment.jsx

```diff
@@ -576,7 +574,7 @@ export default class CreateComment extends React.Component {
                                 value={this.state.message}
                                 onBlur={this.handleBlur}
                                 createMessage={Utils.localizeMessage('create_comment.addComment', 'Add a comment...')}
-                                emojiEnabled={this.state.emojiPickerEnabled}
+                                emojiEnabled={window.mm_config.EnableEmojiPicker === 'true'}
```

Comment:
Okay, so it won't be passed in then? And that's good since I never liked that the FileUpload had to know about that before :)